mempool/src/error/mod.rs

```diff
@@ -61,7 +61,7 @@ pub enum MempoolPolicyError {
     TransactionFeeLowerThanConflictsWithDescendants,
     #[error("Underflow in computing transaction's additional fees.")]
     AdditionalFeesUnderflow,
-    #[error("Transaction does not pay sufficient fees to be relayed.")]
+    #[error("Transaction does not pay sufficient fees to be relayed (tx_fee: {tx_fee:?}, relay_fee: {relay_fee:?}).")]
```

Comment:
This problem seems to apply to many others... for some reason the whole team has been neglecting including the error information in the errors. Everyone, please let's be more strict about this. It helps in debugging.